apps/charterafrica/src/pages/index.page.js

```diff
@@ -42,6 +42,47 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
     fallbackLocale: defaultLocale,
   });
 
+  const { docs: homePage } = await payload.findPage("home", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  const { blocks: pageBlocks } = homePage[0] ?? {
+    blocks: [],
+  };
+
+  const blocks =
+    pageBlocks?.map(({ blockType, ...other }) => ({
+      ...other,
+      slug: blockType,
+    })) || [];
+
+  const spotlight = blocks.find((block) => block.slug === "spotlight");
```

Comment:
```suggestion
  const spotlight = blocks.find((block) => block.slug === "spotlight") || {};
```

On new DB, page won't have `spotlight`.